packages/ui/useScroll/Scrollbar.js

```diff
@@ -50,7 +50,9 @@ const useScrollbar = (props) => {
 	hideScrollbarTrackJob.current = hideScrollbarTrackJob.current || new Job(hideScrollbarTrack, scrollbarTrackHidingDelay);
 
 	function hideScrollbarTrack () {
-		removeClass(scrollbarTrackRef.current, css.scrollbarTrackShown);
+		if (scrollbarTrackRef.current !== null) {
+			removeClass(scrollbarTrackRef.current, css.scrollbarTrackShown);
```

Comment:
How about moving the condition checking `null` into the `removeClass`? If the `removeClass` is called several times, we need to wrap it with condition several times with your code.